src/main/scala/kinesis/mock/api/CreateStreamRequest.scala

```diff
@@ -10,7 +10,7 @@ import kinesis.mock.models._
 import kinesis.mock.syntax.either._
 import kinesis.mock.validations.CommonValidations
 
-final case class CreateStreamRequest(shardCount: Int, streamName: StreamName) {
+final case class CreateStreamRequest(streamName: StreamName, shardCount: Int = 4) {
```

Comment:
This should be an `Option[Int]`, with the logic below providing `4` when the value is not provided. These classes should represent the requests the best that they can. Really not a fan of the default parameter.